codalab/model/bundle_model.py

```diff
@@ -593,6 +593,149 @@ def batch_update_bundles(self, bundles, update, condition=None):
                 return success
         return True
 
+    def set_waiting_for_torque_bundle(self, bundle, torque_job_handle):
```

Comment:
Please rename this and related things to `waiting_for_worker_startup` and remove all references to `torque`.